### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qunabu**
#### Summary
One crucial step, about generating `At Rest Encryption Key` secret was missing from Readme instruction 

#### Ticket Link
https://community-release.mattermost.com/core/pl/4t48e7jifbdnfj1m5dnmbdk69c

### CODE REVIEW

**Review by hanzei — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,7 +73,8 @@ You must first register the Mattermost GitHub Plugin as an authorized OAuth app
 3. Submit.
 4. Click **Generate a new client secret** and provide your GitHub password to continue.
 5. Copy the **Client ID** and **Client Secret** in the resulting screen.
-6. Once you have successfully registered the Mattermost GitHub Plugin as an authorized OAuth app, switch to Mattermost and run `/github connect` in a Mattermost channel. You should receive a DM from the GitHub plugin about the features you have available.
+6. Click on both **Generate** buttons in `Webhook Secret` and `At Rest Encryption Key`
```
Comment (justinegeffen) [inline README.md:76]:
```suggestion
6. Click on both **Generate** buttons in `Webhook Secret` and `At Rest Encryption Key`.
```

File: README.md [hunk 2]
```diff
@@ -73,7 +73,8 @@ You must first register the Mattermost GitHub Plugin as an authorized OAuth app
 3. Submit.
 4. Click **Generate a new client secret** and provide your GitHub password to continue.
 5. Copy the **Client ID** and **Client Secret** in the resulting screen.
-6. Once you have successfully registered the Mattermost GitHub Plugin as an authorized OAuth app, switch to Mattermost and run `/github connect` in a Mattermost channel. You should receive a DM from the GitHub plugin about the features you have available.
+6. Click on both **Generate** buttons in `Webhook Secret` and `At Rest Encryption Key`  
+7. Once you have successfully registered the Mattermost GitHub Plugin as an authorized OAuth app, switch to Mattermost and run `/github connect` in a Mattermost channel. You should receive a DM from the GitHub plugin about the features you have available.
```
Comment (justinegeffen) [inline README.md:77]:
```suggestion
7. Once you've successfully registered the Mattermost GitHub Plugin as an authorized OAuth app, switch to Mattermost and run `/github connect` in a Mattermost channel. You should receive a Direct Message from the GitHub plugin about the features available to you.
```

**Review by justinegeffen — COMMENTED** [review]
Just two suggestions. 

Aside: There are some additional changes that need to be made (replace "click" with "select") but that will be done in a separate PR.

**Review by cwarnermm — APPROVED** [review]
I agree with Justine's suggested updates. Thanks!

**Review by justinegeffen — APPROVED** [review]

**Comment by hanzei** [comment]
@qunabu Would you be open on signing the CLA?

**Comment by qunabu** [comment]
> @qunabu Would you be open on signing the CLA?

sure, can you guide me how?

**Comment by hanzei** [comment]
@qunabu Could you please go to https://mattermost.org/mattermost-contributor-agreement/ and fill out the form?

**Comment by qunabu** [comment]
> @qunabu Could you please go to https://mattermost.org/mattermost-contributor-agreement/ and fill out the form?


done

**Comment by jasonblais** [comment]
Thanks @qunabu for the contribution!

**Pull request merged**